Resources/Prototypes/Entities/Clothing/Back/smuggler_tables.yml

```
@@ -674,3 +676,85 @@
     - id: AbominationCube
       amount: !type:RangeNumberSelector
         range: 1, 10
+
+- type: entityTable #Pirates & Freelancers
+  id: RandomSatchelTable7
+  table: !type:AllSelector #Shit is packed. GearTable first because it causes potential fill issues otherwise.
+    children:
+    - !type:NestedSelector
+      tableId: RandomSatchelPirateGearTable #1x Huge
+    - !type:NestedSelector
+      tableId: RandomSatchelPirateWeaponsTable #1x Small (1x1x1 OR 1x3)
+    - !type:NestedSelector
+      tableId: RandomSatchelPirateClothingTable #1x Medium
+    - !type:NestedSelector
+      tableId: RandomSatchelBootyTable #1x Medium
+
+- type: entityTable #1x Huge
+  id: RandomSatchelPirateGearTable
+  table: !type:GroupSelector
+    children:
+    - id: JetpackBlackFilled
+    - id: ClothingOuterCoatPirate
+    - id: ClothingOuterHardsuitPirateEVA
+    - id: ClothingOuterHardsuitPirateCap
+      weight: 0.5
+
+- type: entityTable #1x Small (1x1x1 OR 1x3)
+  id: RandomSatchelPirateWeaponsTable
+  table: !type:GroupSelector
+    children:
+    - id: Cutlass
+    - id: WeaponRevolverPirateEmpty
+    - id: WeaponPistolFlintlock
+    - id: WeaponRevolverPirate
+      weight: 0.5
+    - id: EnergyCutlass
+      weight: 0.1
+
+- type: entityTable #1x Medium
+  id: RandomSatchelPirateClothingTable
+  table: !type:GroupSelector
+    children:
+    - id: PirateHandyFlag
+    - id: ClothingHeadHatPirate
+    - id: ClothingHeadHatPirateTricord
+    - id: ClothingUniformJumpsuitPirate
+    - id: ClothingNeckCloakPirateCap
+      weight: 0.5
+
+- type: entityTable #1x Medium
+  id: RandomSatchelBootyTable
+  table: !type:GroupSelector
+    children:
+    - id: SpaceCash1000
+      weight: 20
+    - !type:GroupSelector
```

I assume this empty group selector was meant to contain the coins? I'd like to see this group able to roll more than once for the potential to see a mix of coins instead of just the same type. Just reduce the diamond coin to 1-2.
```suggestion
    - !type:GroupSelector
      range: 1, 2
      children:
```